src/components/SwitzerlandEstimatedCasesDivisionModal.tsx

```diff
@@ -151,8 +149,18 @@ export const SwitzerlandEstimatedCasesDivisionModal = ({
           Show cantons
         </span>
       </div>
-      {(!cantonData || !regionData) && <Loader />}
-      {cantonData && regionData && (
+
+      {isLoading && <Loader />}
+      {isError && error && (
```

Comment:
We can simply use `isError` and `error` to display the error messages when there's an error in the fetching of data.